clouddriver-lambda/src/test/java/com/netflix/spinnaker/clouddriver/lambda/deploy/ops/CreateLambdaAtomicOperationTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.lambda.deploy.ops;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+import com.amazonaws.services.lambda.AWSLambda;
+import com.amazonaws.services.lambda.model.CreateFunctionRequest;
+import com.amazonaws.services.lambda.model.CreateFunctionResult;
+import com.amazonaws.services.lambda.model.DeadLetterConfig;
+import com.amazonaws.services.lambda.model.FunctionCode;
+import com.netflix.spinnaker.clouddriver.lambda.deploy.description.CreateLambdaFunctionDescription;
+import org.junit.jupiter.api.Test;
+
+public class CreateLambdaAtomicOperationTest {
+  private CreateLambdaAtomicOperation mockInvoker = mock(CreateLambdaAtomicOperation.class);
+  private CreateFunctionResult conv = mock(CreateFunctionResult.class);
+
+  {
+    when(mockInvoker.operate(any())).thenReturn(mockInvokeResults());
+  }
+
+  private CreateFunctionResult mockInvokeResults() {
```

Comment:
seems like this was part of the old stuff, prob want to clean this guy up as well.

Reply:
Done